components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -13,7 +13,8 @@ import mozilla.components.lib.state.State
  * @property tabs the list of open tabs, defaults to an empty list.
  * @property selectedTabId the ID of the currently selected (active) tab.
  * @property customTabs the list of custom tabs, defaults to an empty list.
- * @property extensions A map of extension ids and web extensions of all installed web extensions.
+ * @property containers A map of session context IDs and their respective container [ContainerState].
```

Comment:
The use of session here makes me think of session IDs. Maybe just say "A map of context IDs" or "A map of [SessionState.contextId]"?

Reply:
Thanks, this is a good change. Will have this resolved in my next revision.